staff/albert-fuente/precourse/Bingo/bingo.js

```diff
@@ -0,0 +1,124 @@
+var arrayUsers=[];  //array where you store the cardBoards
+var bingoCard = [];
+var checkNumbers=[];
+var cardBoard=[
```

Comment:
mejor hacer una función que genere X numeros aleatorios y que no se repitan.
let carton =[]
const rellenaCarton = () => {
  let aleat = 0
  for (let i=0; i<15; i++) {
    do {
      aleat = randomNum()
    } while ( carton.find( item => item === aleat ) )
    carton.push(aleat)
  }
}

Reply:
randomNum() = function que genera numero aleat